packages/aws-cdk-lib/aws-ecs-patterns/test/fargate/load-balanced-fargate-service.test.ts

```diff
@@ -2066,4 +2066,32 @@ describe('NetworkLoadBalancedFargateService', () => {
       },
     });
   });
+
+  test('specify containerCpu And containerMemoryLimitMiB', () => {
```

Comment:
It seems that this test is included in the `NetworkLoadBalancedFargateService` test, so it needs to be moved somewhere within the `ApplicationLoadBalancedFargateService` test.

https://github.com/go-to-k/aws-cdk/blob/ffd9d9ce510a4c820b1437cce93c4772cd7c14c0/packages/aws-cdk-lib/aws-ecs-patterns/test/fargate/load-balanced-fargate-service.test.ts#L15

and nit.

```suggestion
  test('specify containerCpu and containerMemoryLimitMiB', () => {
```

Reply:
https://github.com/aws/aws-cdk/pull/30920/commits/59b2aa1fe39180ed22d597c26e07c881f1c6f610